### ISSUE TITLE
Modal dialogs do not close when they lose focus

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1450]
Some of our modal dialogs do not close after a user tabs off of the last element. The focus has moved back to the underlying page but the modal remains. Make sure the modal closes when it loses focus.

This seems like a common thing that Bootstrap should be addressing. Research bootsrap's docs to determine if we're just missing something. Otherwise add some JS to close/hide the modal on blur.

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'Medium' priority.

**Comment by johnpinto1** [comment]
@sjDCC @xsrust @briri This is no longer an issue as far as I can tell. But a link with a dialog box that opens in a new window can be an issue as it remains visible after clicking link. See screen shots. Not sure if this is worth coreecting, as jasvascript fix might affect some unforseen functionality.


![Selection_056](https://user-images.githubusercontent.com/8876215/69255268-6df24e00-0baf-11ea-96cd-32e8de85b60d.png)
![Selection_054](https://user-images.githubusercontent.com/8876215/69255269-6df24e00-0baf-11ea-87bb-0d7baca8f04c.png)

**Comment by mariapraetzellis** [comment]
@mariapraetzellis will check it and close it if it's fixed.

**Comment by briri** [comment]
ok, I think this is no longer an issue. Here are the places where we use modal dialogs:
- The 'Edit Profile' page - log in, go to the edit profile page, change your email address, click 'Save'. A password confirmation modal dialog will open. When I click off of the modal it disappears properly.
- The 'Edit Plan' page - Edit one of your plans, click on the 'View all' link in the guidance selection area. Click off of the modal dialog. It disappears when you do so. 
- The 'Users' page - Login as an admin, go to the Admin --> Users page, click on 'Edit' for one of the users. The dialog disappears properly when I click off of it.

I also verified that the user can use the tab button to move to the 'cancel' and other buttons on the modal dialog

**Comment by magdalenadrafiova** [comment]
- Edit profile pages 
![image](https://user-images.githubusercontent.com/47178106/69867083-b8558800-129d-11ea-9c59-1f4a9891eebf.png) works well 
- edit plan page works well when I click on expand all and collapse all 
- The 'Users' page - I am not sure @briri how to test this one

**Comment by briri** [comment]
@magdalenadrafiova you can test the 'Users' page by logging in as an admin and the going to the 'Users' page. Then click on the 'Edit' link for one of the users. This will open a modal window that allows you to edit their privileges/roles. If you navigate away from the window (e.g. click on the greyed out page under the modal window) then the modal window properly closes (it was not doing that properly in the past).

**Comment by sjDCC** [comment]
Checked all 3 cases and modals all close. Yay!

Also, the "expand all" button let me tab to guidance embedded in tabs. Couldn't figure out how to do that before
